slack-check-runs-notify/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'slack-check-runs-notify'
+description: 'Check if there is a job that has failed for this commit and report it to Slack'
+author: 'Wunderflats GmbH'
+inputs:
```

Comment:
`GITHUB_TOKEN` is missing here? :thinking:

Reply:
It was 😅